js/src/components/help-popover/index.scss

```diff
@@ -21,14 +21,14 @@ $button-color: #949494;
 	}
 
 	.components-popover__content {
-		width: 300px;
+		width: $gla-size-medium;
 
 		@include break-large {
-			width: 360px;
+			width: $modal-min-width;
```

Comment:
Can you elaborate what do you mean with fragile?

Reply:
Considering it's a popover component, using a global variable that is declared in an external source and named for another type of component makes it look fragile. If its value changes someday, the purpose of the change from external sources is based on its applicability to their modal component, and there is uncertainty as to whether its new value still applies here. Moreover, GLA would not be aware of the change when it occurs.

Reply:
I agree that "modal" suffix is not optimal in terms of lack of flexibility. Kind of the same as discussed here https://github.com/woocommerce/google-listings-and-ads/pull/1578#discussion_r920847312

> If its value changes someday, the purpose of the change from external sources is based on its applicability to their modal component, and there is uncertainty as to whether its new value still applies here. Moreover, GLA would not be aware of the change when it occurs.

I'm not sure about that fragility is an issue since that applies for all the SCSS variables (colors, gaps, spacing, fonts...)  as well as JS components regarding Wordpress and Woo. 

Another "this is not an issue" point is that I see them being used widely over in several places. 

<img width="1326" alt="Screenshot 2022-07-15 at 12 20 36" src="https://user-images.githubusercontent.com/5908855/179183562-af816a63-b117-4151-87a6-f58bd3fc95f1.png">

Reply:
> I'm not sure about that fragility is an issue since that applies for all the SCSS variables (colors, gaps, spacing, fonts...) as well as JS components regarding Wordpress and Woo.

It depends on the association between the names of the external variables and the GLA styles being applied. Popover and modal components are not the same from the perspective of layout, so I thought their association is relatively low.

> Another "this is not an issue" point is that I see them being used widely over in several places.

Sorry, I'm not pretty sure it's a reasonable point. All `$break-*` variables are breakpoints, using them with `@media` syntax is reasonable. But I don't see the values of breakpoints having an association with the dimensional width of a component's layout. Even though breaking points are almost never changed, the semantics are confusing.